Components/modal/modal.css

```diff
@@ -0,0 +1,39 @@
+.modal-box{
+    display: flex;
+    flex-direction: column;
+    background-color: rgb(228, 228, 228);
+    padding: 1.25rem;
+    width: 25rem;
+    border: 1px solid grey;
+    border-radius: 4px;

```

Comment:
i think there is a typo for the color gray here

Reply:
It works with "grey" with my editor. But i'll still make sure its correct, thanks!